deployment/terraform/create.go

```diff
@@ -407,3 +410,24 @@ func (t *Terraform) getOutput() (*terraformOutput, error) {
 	}
 	return &output, nil
 }
+
+func pingServer(addr string) error {
+	mlog.Info("Checking server status:", mlog.String("host", addr))
+	client := model.NewAPIv4Client(addr)
+	client.HttpClient.Timeout = 10 * time.Second
+	timeout := time.After(30 * time.Second)
+
+	for {
+		select {
+		case <-timeout:
+			return fmt.Errorf("timeout after 30 seconds, server is not responding")
+		case <-time.Tick(3 * time.Second):
+			_, resp := client.GetPingWithServerStatus()
+			if resp.Header.Get("database_status") == "OK" {
```

Comment:
We should also check if `resp.Err != nil`. Let's get to the bottom of the filestore issue also.